rc4me/util_test.py

```diff
@@ -3,11 +3,6 @@
 from rc4me.run import cli
 
 
-def test_pass():
-    """ Temp test before any written"""
-    pass
-
-
 def check_repo_files_in_home(repo: Path):
     """Check that files in a repo are in the rc4me destination dir."""
     home_rcs = [f.name for f in Path("~").expanduser().glob(".*")]
```

Comment:
Should this be the same as `dest = Path.home()`. Maybe we should do a global so these don't diverge.

Reply:
No longer needed after #40